packages/migrate/migrations/svelte-4/index.js

```diff
@@ -25,22 +25,49 @@ export async function migrate() {
 		process.exit(1);
 	}
 
+	const migrate_transition = await prompts({
+		type: 'confirm',
+		name: 'value',
+		message:
+			'Add the `|global` modifier to currently global transitions for backwards compatibility? More info at https://svelte.dev/docs/v4-migration-guide#transitions-are-local-by-default',
+		initial: true
+	});
+
+	const folders = await prompts({
+		type: 'multiselect',
+		name: 'value',
+		message: 'Which folders should be migrated?',
+		choices: fs
+			.readdirSync('.')
+			.filter(
+				(dir) => fs.statSync(dir).isDirectory() && dir !== 'node_modules' && !dir.startsWith('.')
+			)
+			.map((dir) => ({ title: dir, value: dir, selected: true }))
+	});
+
+	if (!folders.value?.length) {
+		process.exit(1);
+	}
+
+	update_pkg_json();
+
 	// const { default: config } = fs.existsSync('svelte.config.js')
 	// 	? await import(pathToFileURL(path.resolve('svelte.config.js')).href)
 	// 	: { default: {} };
 
 	/** @type {string[]} */
-	const svelte_extensions = /* config.extensions ?? - disabled because it would break .svx */ ['.svelte'];
+	const svelte_extensions = /* config.extensions ?? - disabled because it would break .svx */ [
+		'.svelte'
+	];
 	const extensions = [...svelte_extensions, '.ts', '.js'];
-	// TODO read tsconfig/jsconfig if available? src/** will be good for 99% of cases
-	const files = glob('src/**', { filesOnly: true, dot: true }).map((file) =>
-		file.replace(/\\/g, '/')
-	);
+	const files = glob(`{${folders.value.join(',')}}/**`, { filesOnly: true, dot: true })
+		.map((file) => file.replace(/\\/g, '/'))
```

Comment:
Is this necessary? I think glob uses posix format by default.

Reply:
Better safe than sorry, I vaguely remember this being necessary